_posts/2019-07-09-ranking-factors.md

```diff
@@ -45,20 +45,22 @@ Note: Sites using the search results API to present our results on their own web
 
 ### Core Ranking Algorithm
 
-Our system is built on Elasticsearch, which itself is built on Apache Lucene. For the first several generations, Elasticsearch used Lucene’s default ranking, the Practical Scoring Function. This Function starts with a basic Boolean match for single terms and adds in TF/IDF and a vector space model. Here are some high level definitions for these technical terms:
+Our system is built on Elasticsearch, which itself is built on Apache Lucene. 
```

Comment:
This edit changed the meaning in an unexpected way. I want to maintain the differentiation between Lucene's features and Elasticsearch's features, for the people who say "isn't it just Lucene?"

I suggest making the first sentence "Our system is built on Elasticsearch, which itself is built on Apache Lucene, and over time they have supplemented Lucene's Practical Scoring Function approach in a few ways."